src/Service/PartyHelper.php

```diff
@@ -281,6 +281,11 @@ public function getRelevantPartiesForHearingPostByCase(CaseEntity $case): array
             $complainants[$relation->getParty()->getName().', '.$relation->getType()] = $relation->getParty();
         }
 
+        // Sort alphabetically
+        if (count($complainants) > 1) {
+            uasort($complainants, static fn ($a, $b) => strcmp($a->getName(), $b->getName()));
+        }
```

Comment:
Skip the check for non-empty array (the jit compiler will probably optimize).

You can use the spaceship operator, `<=>`, rather than `strcmp`: `static fn ($a, $b) => $a->getName() <=> $b->getName()`. The spaceship operator can handle other types than string so it will work when sorting on other types as well.

Reply:
Removed check and used spaceship operator.